### PR TITLE
Typo docker-machine -f is shortcut for --format

### PR DESCRIPTION
**Pull request by mishak87**

### CODE REVIEW

**Comment by dgageot** [comment]
Thank you @mishak87. You have to sign your commits for us to merge them.

**Comment by mishak87** [comment]
@dgageot Done. I used web then I remembered I have to sign.

**Comment by dgageot** [comment]
LGTM. ping @docker/core-machine-maintainers

File: docs/reference/ls.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ the -t flag for this purpose with a numerical value in seconds.
 
 ## Filtering
 
-The filtering flag (`-f` or `--filter)` format is a `key=value` pair. If there is more
+The filtering flag (`--filter)` format is a `key=value` pair. If there is more
```
Comment (nathanleclaire) [inline docs/reference/ls.md]: Looks like the first backtick includes the close paren (`)`) as well :(

**Comment by dgageot** [comment]
Good catch!

**Comment by dgageot** [comment]
My brain wanted to read it right!

**Comment by nathanleclaire** [comment]
:)  @mishak87 if you fix the rogue paren LGTM

**Comment by mishak87** [comment]
@nathanleclaire Haha nice catch didn't notice that one :-)
If you knew what I had to do to produce that commit :smile:.

*As you might know Division is in Open Beta today I am on bad network trying to download it. That means no reboot to linux to save precious download time. Working with Windows I boot into once in a month. I had to download GitHub for Desktop to sign that commit (download failed three times because .NET installer has no concept of checksums and Range requests); Atom to edit that bracket. I just pray you don't ask me to squash them together.  
In hindsight I should have just switched to linux and be done with it in 2 minutes.*

**Comment by nathanleclaire** [comment]
> In hindsight I should have just switched to linux and be done with it in 2 minutes.

Haha normally I would but due to your Herculean effort I think I can make an exception.

LGTM

**Pull request merged**